characterEntities/YellowHero.java

```diff
@@ -86,4 +56,28 @@ protected boolean isHit(Ability ability, Entity target) {
 		}
 		return hit;
 	}
+
+	@Override
+	public void update() {
+		super.update();
```

Comment:
All the heroes have the same default ability so it's currently repetitive. This function will be different across the heroes in the future